__tests__/app.test.js

```diff
@@ -85,22 +85,20 @@ describe('gitty routes', () => {
     });
   });
 
-  it('shows user all posts', async () => {
-    const agent = request.agent(app);
+  it('should login and redirect user to posts', async () => {
+    const res = await request
+      .agent(app)
+      .get('/api/v1/github/login/callback?code=42')
+      .redirects(1);
 
-    await UserService.create({
+    expect(res.status).toEqual(200);
+    expect(res.body).toEqual({
+      id: expect.any(String),
       username: 'fake_github_user',
       email: 'not-real@example.com',
+      // password: 'password',
+      iat: expect.any(Number),
+      exp: expect.any(Number),
```

Comment:
i thought i was doing everything wrong, and then realized that I already wrote this same exact test up above, so I stashed this commit, and ended up fixing the original 1 I had..